src/main/java/uk/gov/hmcts/ccd/data/casedetails/query/CaseDetailsQueryBuilder.java

```diff
@@ -81,6 +82,13 @@ public CaseDetailsQueryBuilder whereState(String state) {
         return this;
     }
 
+    public CaseDetailsQueryBuilder whereStates(List<String> states) {
+        if (CollectionUtils.isNotEmpty(states)) {
```

Comment:
can we use
```if(! states.isEmpty())```
?

Reply:
Any reason for not using apache commons CollectionUtils ?

Reply:
just because this can be done using no library method and is even more readable(for me of course)  :)
leave it if you want no big deal

Reply:
just that it checks for null and empty collection in one call, I tend to use it :-)